tests/dummy_backend.py

```diff
@@ -44,9 +44,9 @@ def browse(self, path):
     def get_distinct(self, field, query=None):
         return self.dummy_get_distinct_result.get(field, set())
 
-    def lookup(self, uri):
-        uri = Ref.track(uri=uri).uri
-        return [t for t in self.dummy_library if uri == t.uri]
+    def lookup_many(self, uris):
+        uris = [Ref.track(uri=uri).uri for uri in uris]
```

Comment:
I see that this comes from the old implementation, but what is this line needed for? :thinking: 

```suggestion
    def lookup_many(self, uris):
```

Reply:
oops I think it was either an artifact from some debugging or maybe from a rebase! will commit the deletion